From dd6759afb513abd956c907f4cfe329a331c74a37 Mon Sep 17 00:00:00 2001
From: Alexandre Lissy <alissy@mandriva.com>
Date: Sun, 1 Apr 2012 15:05:35 +0200
Subject: [PATCH] fix: Force link of libOGDF.so against pthread

OGDF uses pthread, and we have to enforce the link against pthread ...
---
 thirdparty/OGDF/CMakeLists.txt |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/thirdparty/OGDF/CMakeLists.txt b/thirdparty/OGDF/CMakeLists.txt
index 30e0433..ae88ef1 100755
--- a/thirdparty/OGDF/CMakeLists.txt
+++ b/thirdparty/OGDF/CMakeLists.txt
@@ -19,7 +19,11 @@ IF(WIN32)
   TARGET_LINK_LIBRARIES(${OGDFLibrary} psapi)
 ENDIF(WIN32)
 
+IF(UNIX)
+  TARGET_LINK_LIBRARIES(${OGDFLibrary} pthread)
+ENDIF(UNIX)
+
 INSTALL(TARGETS ${OGDFLibrary}
        RUNTIME DESTINATION ${TulipBinInstallDir} COMPONENT tulip_ogdf
        LIBRARY DESTINATION ${TulipLibInstallDir} COMPONENT tulip_ogdf
-       ARCHIVE DESTINATION ${TulipLibInstallDir} COMPONENT tulip_ogdf)
\ No newline at end of file
+       ARCHIVE DESTINATION ${TulipLibInstallDir} COMPONENT tulip_ogdf)
-- 
1.7.9.2

